org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4whole/PreferencesDialog.java

```diff
@@ -74,6 +80,10 @@
 import edu.mit.csail.sdg.alloy4.Subprocess;
 import edu.mit.csail.sdg.translator.A4Options.SatSolver;
 
+/**
+ * @modified: Nuno Macedo // [HASLab] electrum-base, electrum-unbounded,
+ *            electrum-decomposed
```

Comment:
added SMV backend solvers (packaged as an executable binary); added decomposed solving strategy to options